changelog.txt

```diff
@@ -13,6 +13,15 @@
 * Add logger connector which allows to override logger behaviour. (#796)
 * Include redirect uri in body when redeeming refresh token at token endpoint (#815)
 
+Version 1.5.5
```

Comment:
Is the version expected to be here?

Reply:
similar comment addressed in MSAL codebase.